lib/compiler/Cargo.toml

```diff
@@ -21,16 +21,18 @@ serde = { version = "1.0", features = ["derive"], optional = true }
 thiserror = "1.0"
 serde_bytes = { version = "0.11", optional = true }
 smallvec = "1.6" 
+rkyv = { version = "0.4.1", optional = true }
 
 [features]
-default = ["std", "enable-serde"]
+default = ["std", "enable-serde", "enable-rkyv"]
 # This feature is for compiler implementors, it enables using `Compiler` and
 # `CompilerConfig`, as well as the included wasmparser.
 # Disable this feature if you just want a headless engine.
 translator = ["wasmparser"]
 std = ["wasmer-types/std"]
 core = ["hashbrown", "wasmer-types/core"]
 enable-serde = ["serde", "serde_bytes", "wasmer-types/enable-serde"]
+enable-rkyv = ["rkyv"]
```

Comment:
Shouldn't they be exclusive? Either `enable-serde` or `enable-rkyv`?

Reply:
It is not, because structs defined in lib/compiler are shared in all artifacts, but only native artifact switched to use rkyv to deserialize